recipe-client-addon/lib/SandboxManager.jsm

```diff
@@ -65,6 +65,43 @@ this.SandboxManager = class {
       }
     });
   }
+
+  /**
+   * Wraps a function that returns a Promise from a privileged (e.g. chrome)
```

Comment:
> 'privileged' and 'chrome' are the same thing,

I thought they were subtly different things (are there other non-chrome contexts that can be considered privileged?), hence the e.g. I'll fix this.